website/docs/blocks.md

```diff
@@ -127,10 +133,24 @@ JSON-RPC
 }
 ```
 
+```
+JSON-RPC
+{
+    "jsonrpc": "2.0",
+    "id": "1",
+    "method": "devnet_abortBlocks",
+    "params": {
+        "starting_block_id": BLOCK_ID
+    }
+}
+```
+
 Response:
 
 ```
 {
     "aborted": [BLOCK_HASH_0, BLOCK_HASH_1, ...]
 }
 ```
+
+If both `starting_block_hash` and `starting_block_id` are passed, error is thrown. Note: param `starting_block_hash` is being depreciated as block hash is contained in `BLOCK_ID` object.
```

Comment:
```suggestion
If both `starting_block_hash` and `starting_block_id` are passed, error is thrown. Note: param `starting_block_hash` is being deprecated as block hash is contained in `BLOCK_ID` object.
```

Reply:
Please use `[skip ci]` in the commit message when applying documentation changes. This is to prevent CircleCI from executing the whole workflow and incurring unnecessary cost.

[Docs](https://circleci.com/docs/skip-build/#skip-jobs)